sig-scalability/annual-report-2021.md

```diff
@@ -4,79 +4,66 @@
 
+3. KEP work in 2021:
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - None
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [1040 - Priority and Fairness for API Server Requests](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1040-priority-and-fairness/README.md) - 1.23
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - [647 - APIServer Tracing](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/647-apiserver-tracing/README.md) - 1.22
+     - [1669 - Proxy Terminating Endpoints](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1669-proxy-terminating-endpoints/README.md) - 1.22
+     - [2464 - Kubetest2 CI Migration](https://github.com/kubernetes/enhancements/blob/master/keps/sig-testing/2464-kubetest2-ci-migration/README.md) - 1.21
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - None
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Each subproject could benefit from additional hands. However, the Scalability Test Frameworks and Scalability and Performance tests and validation are the ones where we can grow contributors - the other require both very deep and wide understanding of Kubernetes before making reasonable contributions.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We care mostly about metrics for our main repository [perf-tests]. We measure them through [devstats]. We keep track of number of reviewers, opened issues and PRs merged.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - Yes, [CONTRIBUTING.md] points to maintained list of issues with label `good first issue`.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We don't have any special requirements for reviewers/approvers.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, we have contributors from multiple companies.
```

Comment:
```suggestion
   - Yes, we have contributors from [multiple companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Scalability&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609477200000&to=1640926800000).
```
Not a blocker by any means, but this will link out to devstats showing the different companies for 2021 :)